lib/optimizely/project_config.rb

```diff
@@ -451,6 +460,16 @@ def get_rollout_from_id(rollout_id)
       nil
     end
 
+    def feature_experiment?(experiment_id)
+      # Determines if a given experiment ID belongs to any Feature.
```

Comment:
nit. `Determines if a given experiment ID is related to a feature.` or `Determines if given experiment is a feature test.`